staging/src/k8s.io/kube-aggregator/pkg/apis/apiregistration/types.go

```diff
@@ -52,7 +52,7 @@ type APIServiceSpec struct {
 	// InsecureSkipTLSVerify disables TLS certificate verification when communicating with this server.
 	// This is strongly discouraged.  You should use the CABundle instead.
 	InsecureSkipTLSVerify bool
-	// CABundle is a PEM encoded CA bundle which will be used to validate an API server's serving certificate.
+	// CABundle is a PEM(Base64) encoded CA bundle which will be used to validate an API server's serving certificate.
```

Comment:
drop this change, and add "The value should be base64 encoded when specified in JSON or YAML." at the end.

also, change the apiregistration/v1/types.go and apiregistration/v1beta1/types.go files, or the doc won't make it into the generated API docs